ajax/widget/js/widget.js

```diff
@@ -0,0 +1,40 @@
+let employeeList = new XMLHttpRequest();
+
+employeeList.onreadystatechange = () => {
+    if (employeeList.readyState === 4) {return;}
+    let employees = JSON.parse(employeeList.responseText);
+    let employeeStatus = '<ul class="bulleted">';
+    for (let i = 0; i < employees.length; i++) {
+        let employeeStatusClass = "out";
+        if (employees[i].inoffice === true) {
+            employeeStatusClass = 'in';
+        }
+        employeeStatus += `<li class="${employeeStatusClass}">
+        ${employees[i].name}</li>`;
```

Comment:
This looks weird  to me on two lines...
Can you put that on one line.

Reply:
Done.